packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -365,7 +369,11 @@ export function logBlockingStart(
       reusableLaneOptions.start = updateTime;
       reusableLaneOptions.end = renderStartTime;
       performance.measure(
-        isSpawnedUpdate ? 'Cascade' : 'Blocked',
+        isSpawnedUpdate
+          ? 'Cascading Update'
+          : renderStartTime - updateTime > 5
+            ? 'Update Blocked'
+            : 'Update',
```

Comment:
I was thinking "Scheduled" which also makes sense since it's the Scheduler track

Reply:
It's tricky because the time span really mean that but it actually doubles as a "mark" too in the beginning which is the update. It's just that marks are kind of useless in this UI since they 1) stack on top of the span 2) are impossible to click. Where as the span is easier to see and click. Even if it's very small.